src/jit/gentree.h

```diff
@@ -5010,12 +5061,11 @@ inline var_types& GenTree::CastToType()
 // Notes:
 //    Of the block ops only GT_COPYOBJ is allowed to have GC pointers.
```

Comment:
Yes, I'll fix it to say "Of the block nodes, only GT_OBJ and ST_STORE_OBJ are allowed to have GC pointers."

---
In reply to: [77273635](https://github.com/dotnet/coreclr/pull/7027#discussion_r77273635) [](ancestors = 77273635)